### PR TITLE
Portfolio - mobile menu

### PR DESCRIPTION
**Pull request by naji-97**
- [ ] When the user clicks (or taps) the hamburger button, the mobile menu appears.
- [ ] When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- [ ] When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- [ ] When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

**Comment by mirouhml** [comment]
Hi @naji-97 @AbdelazizSoliman, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a complete README, please know that all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present, the only optional section is FAQ, also, make sure to add the section Future Features, including 1-3 potential features to be added, also, please take care of the styling issues you have in your README some texts aren't properly aligned and such, I recommend that you copy the template as it is then start editing it out.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 1]
```diff
@@ -12,19 +12,31 @@
...
     <header>
         <div class="navbar flex">
             <a class="logo" href="#"> My logo</a>
             <ul class="links">
-                <li><a href="#">Protfolio</a> </li>
-                <li><a href="#">About</a> </li>
-                <li><a href="#">Contact</a></li>
+                <li><a href="#Protfolio">Protfolio</a> </li>
+                <li onclick="mobileMenu()"><a href="#about">About</a> </li>
+                <li><a href="#Contact">Contact</a></li>
```
Comment (joskalenda) [inline index.html]: - [ ] Well done on creating the nav menu, but I have some issues raised before the project get approved, take a look at this section below, ⬇️ 

-  the spaces between the close icon and the nav items are not the same as in the original, 
- The `font weight`, `spaces`, and `font-size` of your nav items are not the same as the ones on the original design.

File: index.html [hunk 2]
```diff
@@ -12,19 +12,31 @@
...
+            </ul>
+            <div class="menu"><i onclick="mobileMenu()" class="fa-solid fa-bars"></i></div>
+        </div>
+        <div class="mobile-menu non-display" id="mobile-menu">
```
Comment (joskalenda) [inline index.html]: - [ ] Take a look at this record down here, guess what is happening? when the user scrolls this happens ⬇️ 
- The nav menu is not fixed, Please make a few changes to fix this.

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @naji-97 , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ].. ✅ Good design
- [ ].. ✅ Great commit messages 
- [ ].. ✅ Green linters

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @naji-97,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**